src/api/spec/models/user_spec.rb

```diff
@@ -21,6 +21,36 @@
     it { expect(create(:user)).to validate_uniqueness_of(:login).with_message('is the name of an already existing user') }
   end
 
+  describe '#is_active?' do
+    it 'returns true if user is confirmed' do
+      confirmed_user = create(:confirmed_user, login: 'user')
```

Comment:
why is that not in a `let`? :see_no_evil:

Reply:
Actually, I used a handful of  `let`s  in the `describe` of this test,  but I changed them before making the commit because it seemed to me more explicit and less "noisy" in that way.

Now, after review it to accomplish your requested changed, I think that I found a better way, putting extra needed lets at the top, in the main `describe`.

Indeed, looks better.